pkg/registry/core/chain/ns_registry.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 Cisco Systems, Inc.
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Normal etiquette here is to add the new copyright rather than replacing the existing one.  In this case however, as there's basically nothing meaningful left of the original file... this seems fine to me.

Reply:
oops, sorry, will fix!

Reply:
@edwarnicke you can check it here https://github.com/networkservicemesh/sdk/tree/master/pkg/registry/core/chain

Reply:
but will fix :)

Reply:
Done